### ISSUE TITLE
Clean project with debezium-quarkus-outbox fails with BuildException

### ISSUE THREAD

**Issue by rsvoboda** [quarkusio/quarkus#11820]
Clean project with debezium-quarkus-outbox fails with BuildException.
There are no code changes, just generated skeleton application.

```
java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkus.arc.deployment.ArcProcessor#validate threw an exception: javax.enterprise.inject.spi.DeploymentException: javax.enterprise.inject.UnsatisfiedResolutionException: Unsatisfied dependency for type javax.persistence.EntityManager and qualifiers [@Default]
	- java member: io.debezium.outbox.quarkus.internal.EventDispatcher#entityManager
	- declared on CLASS bean [types=[io.debezium.outbox.quarkus.internal.EventDispatcher, java.lang.Object], qualifiers=[@Default, @Any], target=io.debezium.outbox.quarkus.internal.EventDispatcher]
	at io.quarkus.arc.processor.BeanDeployment.processErrors(BeanDeployment.java:990)
	at io.quarkus.arc.processor.BeanDeployment.init(BeanDeployment.java:234)
	at io.quarkus.arc.processor.BeanProcessor.initialize(BeanProcessor.java:122)
	at io.quarkus.arc.deployment.ArcProcessor.validate(ArcProcessor.java:391)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:932)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2046)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1578)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
	at java.base/java.lang.Thread.run(Thread.java:834)
	at org.jboss.threads.JBossThread.run(JBossThread.java:479)
Caused by: javax.enterprise.inject.UnsatisfiedResolutionException: Unsatisfied dependency for type javax.persistence.EntityManager and qualifiers [@Default]
	- java member: io.debezium.outbox.quarkus.internal.EventDispatcher#entityManager
	- declared on CLASS bean [types=[io.debezium.outbox.quarkus.internal.EventDispatcher, java.lang.Object], qualifiers=[@Default, @Any], target=io.debezium.outbox.quarkus.internal.EventDispatcher]
	at io.quarkus.arc.processor.Beans.resolveInjectionPoint(Beans.java:494)
	at io.quarkus.arc.processor.BeanInfo.init(BeanInfo.java:362)
	at io.quarkus.arc.processor.BeanDeployment.init(BeanDeployment.java:226)
	... 14 more
```

Reproducer:
```
mvn io.quarkus:quarkus-maven-plugin:1.7.1.Final:create \
    -DprojectGroupId=org.acme \
    -DprojectArtifactId=getting-started \
    -DclassName="org.acme.getting.started.GreetingResource" \
    -Dpath="/hello" \
    -Dextensions="quarkus-outbox"
cd getting-started
mvn package
```

**Comment by geoand** [comment]
I am almost certain this has been fixed in the latest debezium release.

cc @gunnarmorling @gsmet

**Comment by gsmet** [comment]
Yeah this is known, will be fixed with the upgrade to 1.2.3.Final.
